### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by blackgirlbytes**
This is a proposed pull request template to make our repository more friendly and help us receive quality PRs during Hacktoberfest.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- [ ] 📜 readme
+- [ ] 📜 contributing.md
+- [ ] 📓 general documentation
+- [ ] 🙅 no documentation needed
+
+## [optional] Are there any post-deployment tasks we need to perform?
+
+
+
+## [optional] What gif best describes this PR or how it makes you feel?
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:69]: Love it!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+-->
+Resolves #
+
+## Mobile & Desktop Screenshots/Recordings
+
+<!-- Visual changes require screenshots -->
+
+## Added code snippets?
+- [ ] 👍 yes
+- [ ] 🙅 no, because they aren't needed
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:50]: How about an optional field here for notes/detail?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Please use this format link issue numbers: Resolves #123
+https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
+-->
+Resolves #
+
+## Mobile & Desktop Screenshots/Recordings
+
+<!-- Visual changes require screenshots -->
+
+## Added code snippets?
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:48]: YES, thank you for encouraging great testing

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- [ ] 👍 yes
+- [ ] 🙅 no, because they aren't needed
+
+## Added tests?
+
+- [ ] 👍 yes
+- [ ] 🙅 no, because they aren't needed
+- [ ] 🙋 no, because I need help
+
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:57]: Same as above - optional notes/detail field here about why the dev believes tests aren't neeed?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+## Added to documentation?
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:58]: You're on fire - YES to encouraging docs updates in the cases the code changes warrant it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
+  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
+  
+  For a timely review/response, please avoid force-pushing additional
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Very nice guidance on force-pushing

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+<!--
+  For Work In Progress Pull Requests, please use the Draft PR feature,
```
Comment (ALRubinger) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Awesome to note the drafts feature

**Review by ALRubinger — APPROVED** [review]
Approved pending @blackgirlbytes considering the suggestions laid out in line comments

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (blackgirlbytes) [reply .github/PULL_REQUEST_TEMPLATE.md:57]: done!

**Pull request merged**